docker-compose.debug.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.4"
+
+services:
+  polis:
+    image: polis
+    build:
+      context: server
+      dockerfile: ./Dockerfile
+    environment:
+      NODE_ENV: development
+
+    command: ["npm", "run", "dev"]
```

Comment:
Can the intention of this new docker-compose file be documented? I'm not sure I understand it's use-case